engineering/conventions.md

```diff
@@ -63,6 +64,35 @@ Some projects might define both build and runtime requirements that go beyond wh
 
 * If it is idiomatic in your language, prefer early handling of errors and return early (more info: [Avoid Else, Return Early](http://blog.timoxley.com/post/47041269194/avoid-else-return-early), [GuardClause](http://wiki.c2.com/?GuardClause) and [HandleErrorsInContext](http://wiki.c2.com/?HandleErrorsInContext)).
 
+## TODO comments
```

Comment:
```suggestion
## Keyword Comments
```

Reply:
To be hones, I would have skipped this section of the guide if I've seen "keyword comments". The reason is that it sounds like a convention about `@arg` comments or something like this, so not relevant to Go.

I think it's clear what "TODO comments" are for anyone. `FIXME` may also be considered a "TODO comment", but that's my personal opinion, of course :)

Reply:
That's fair. What I find confusing, though, is to have a section called "TODO comments" that then proceeds to define two different flavours (one of which does and the other of which does not use the word "TODO") in terms of an abstract keyword. 

I don't insist we call them "keyword comments", but maybe we should just define TODO and FIXME each under their own headings (since I don't really foresee adding more keywords later).

Reply:
Renamed the section to mention both keywords.